tensorflow_datasets/image/deep_weeds.py

```diff
@@ -98,19 +101,38 @@ def _split_generators(self, dl_manager):
             url=_URL,
             extract_method=tfds.download.ExtractMethod.ZIP))
 
+
+    path_lbl = dl_manager.download_and_extract(
+        tfds.download.Resource(
```

Comment:
Please update the `fake_data/` dir to add this new file.

https://github.com/tensorflow/datasets/tree/master/tensorflow_datasets/testing/test_data/fake_examples/deep_weeds

Make sure to re-run the `deep_weed_test.py` to make sure the tests are passing.
You may have to move the fake images to a separate `images/` folder and add a `DL_EXTRACT_RESULT` to the `DeepWeedsTest `:

```python
DL_EXTRACT_RESULT = {
      "image": "images/",
      "label": "labels.csv",
}
```